[2023-08-28T16:07:04.099+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tbf_analysis_dag_final.authenticate_service_acct_task manual__2023-08-28T16:07:01.709445+00:00 [queued]>
[2023-08-28T16:07:04.104+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tbf_analysis_dag_final.authenticate_service_acct_task manual__2023-08-28T16:07:01.709445+00:00 [queued]>
[2023-08-28T16:07:04.104+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-08-28T16:07:04.111+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): authenticate_service_acct_task> on 2023-08-28 16:07:01.709445+00:00
[2023-08-28T16:07:04.116+0000] {standard_task_runner.py:57} INFO - Started process 317 to run task
[2023-08-28T16:07:04.120+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'tbf_analysis_dag_final', 'authenticate_service_acct_task', 'manual__2023-08-28T16:07:01.709445+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/tbf_analysis_dag_final.py', '--cfg-path', '/tmp/tmpcyjyz71a']
[2023-08-28T16:07:04.124+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask authenticate_service_acct_task
[2023-08-28T16:07:04.153+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-28T16:07:04.194+0000] {task_command.py:415} INFO - Running <TaskInstance: tbf_analysis_dag_final.authenticate_service_acct_task manual__2023-08-28T16:07:01.709445+00:00 [running]> on host 64e6482d77b4
[2023-08-28T16:07:04.254+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tbf_analysis_dag_final' AIRFLOW_CTX_TASK_ID='authenticate_service_acct_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T16:07:01.709445+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-28T16:07:01.709445+00:00'
[2023-08-28T16:07:04.256+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-28T16:07:04.260+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo starting_gcloud_auth; gcloud auth activate-service-account terraform@trans-border-freights-394419.iam.gserviceaccount.com --key-file /.google/credentials/google_credentials.json; echo ending_gcloud_auth']
[2023-08-28T16:07:04.269+0000] {subprocess.py:86} INFO - Output:
[2023-08-28T16:07:04.270+0000] {subprocess.py:93} INFO - starting_gcloud_auth
[2023-08-28T16:07:05.207+0000] {subprocess.py:93} INFO - Activated service account credentials for: [terraform@trans-border-freights-394419.iam.gserviceaccount.com]
[2023-08-28T16:07:05.766+0000] {subprocess.py:93} INFO - ending_gcloud_auth
[2023-08-28T16:07:05.767+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-28T16:07:05.797+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=tbf_analysis_dag_final, task_id=authenticate_service_acct_task, execution_date=20230828T160701, start_date=20230828T160704, end_date=20230828T160705
[2023-08-28T16:07:05.834+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-28T16:07:05.860+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
